pymc3/step_methods/metropolis.py

```diff
@@ -283,6 +295,122 @@ def competence(var):
             return Competence.IDEAL
         return Competence.INCOMPATIBLE
 
+class CategoricalGibbsMetropolis(ArrayStep):
+    """Metropolis-Hastings optimized for categorical variables. This step
```

Comment:
Also add info that this is doing "Gibbs" -- i.e. updating each dimension one at a time, rather than blocked.